policy/modules/contrib/hg.te

```diff
@@ -0,0 +1,65 @@
+policy_module(hg, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+## <desc>
+##	<p>
+##	Determine whether Mercurial CGI
+##	can access cifs file systems.
+##	</p>
+## </desc>
+gen_tunable(hg_cgi_use_cifs, false)
+
+## <desc>
+##	<p>
+##	Determine whether Mercurial CGI
+##	can access nfs file systems.
+##	</p>
+## </desc>
+gen_tunable(hg_cgi_use_nfs, false)
+
+apache_content_template(hg)
+apache_content_alias_template(hg, hg)
```

Comment:
Are you sure you need also this template?

Reply:
Every other policy that uses apache_content_template followed with it, and it wasn't happy without it at the time I created the policy